src/algorithms/MathUtil.java

```diff
@@ -60,10 +57,16 @@ public static int abs(int n) {
      * @return reversed number
      */
     public static int reverse(int number) {
```

Comment:
Hakob jan, this is a working solution but,  please try to implement it without changing the type to String.

Reply:
Done. I have changed the method.